web/public/js/directives/cd-picture-grid/index.js

```diff
@@ -3,8 +3,37 @@
 
   angular.module('cpZenPlatform').component('cdPictureGrid', {
     bindings: {
-      items: '<'
+      items: '<',
+      actions: '<',
+      multipleSelection: '<'
     },
-    templateUrl: '/directives/tpl/cd-picture-grid'
+    templateUrl: '/directives/tpl/cd-picture-grid',
+    controller: function ($element) {
+      var ctrl = this;
+      ctrl.showActionBar = false;
+      ctrl.selectedItems = [];
+      var gridItems = null;
+      var selectedClass = 'cd-picture-grid__item--selected';
+
+      ctrl.handleSelection = function (e, item) {
+        if (ctrl.actions) {
+          e.preventDefault();
+          var $el = $(e.currentTarget);
+          if (ctrl.multipleSelection !== true && ctrl.selectedItems.length > 0) {
+            gridItems = gridItems || $element.find('.cd-picture-grid__item'); // so we only query the DOM once
```

Comment:
eh, whatever, i was just thinking that the lack of refresh was blocking it, but since we're paginated everywhere, we'll consider it's only page/page.
Will it refresh the "selection" post-new page?